geoips/interfaces/base.py

```diff
@@ -254,7 +254,7 @@ def plugin_repr(obj):
     return f'{obj.__class__}(name="{obj.name}", module="{obj.module}")'
 
 
-def plugin_module_to_obj(name, module, obj_attrs={}):
+def plugin_module_to_obj(cls, name, module, obj_attrs={}):
```

Comment:
I think we should move this function onto the BaseModuleInterface class in the same way that plugin_yaml_to_obj is attached to BaseYamlInterface. It should also be a classmethod.

Reply:
This issue will also fix your comments below.

Reply:
I'll create a separate branch for this. Don't want to include too many changes in this branch, and it will be useful to have this separated just in case something goes wrong and we need to accurately track/source those errors.

Reply:
https://github.com/NRLMMD-GEOIPS/geoips/pull/369